master/networking/vxlan-ipip.md

```diff
@@ -4,11 +4,13 @@ title: Connecting workloads across networks that you do not control
 
 ### Big Picture
 
-Enable inter workload communication across networks that are not workload aware.
+Enable inter workload communication across networks that are not aware of workload IPs.
 
 ### Value
 
-Calico supports encapsulation so you can send traffic between workloads, without requiring the underlying network to be aware of workload IP addresses.
+We generally recommend running Calico without network overlay/encapsulation.  This gives you the highest performance and simplest network; the packet that leaves your workload is the packet that goes on the wire.  
+
+However, selectively using overlays/encapsulation can be useful when running on top of an underlying network that cannot easily be made aware of workload IPs.  A common example is if you are using Calico networking in AWS across multiple VPCs/subnets.  In this case, Calico can selectively encapsulate only the traffic that is routed between the VPCs/subnets, and run without encapsulation within each VPC/subnet. You might also decide to run your entire Calico network with encapsulation as an overlay network -- as a quick way to get started without setting up BGP peering or other routing information in your underlying network.
```

Comment:
This makes it sound like you can do cross-subnet with any of our overlay technologies but VXLAN doesn't support it (yet).  Maybe we should explicitly call that out as a note since the absence of a cross-subnet VXLAN section looks like a mistake.

Reply:
Regarding VXLAN IPIP...it was by design to focus How Tos on what user wants to accomplish rather than technology, implementations. The title was approved by Alex and Casey so I don't see it changing. In this particular case, I see your point that these are key words. From my limited experience, I think our search is quite good. If users are thinking VXLAN/ IP IP, they will still get to this page.

Reply:
I discussed VXLAN/cross subnet with Alex. In this case, the preference is to explicitly state what we do support for cross subnet (so I adjusted the text to make it more explicit). However, Alex and I will need continued discussions on the matter of supported/not supported and inline versus more global page of product matrix support. Let's agree for now that we are not consistent, and we need to provide better guidance in the future because this will come up again.